xios_examples/read_domain_resample/README

```diff
@@ -0,0 +1,4 @@
+read_domain_resample
+--------------------
+
+This example reads in a sample NetCDF file of arbitrary data on a 5 x 5 rectilinear (structured) horizontal domain and resamples to a 4 x 4 rectilinear horizontal domain using bilinear interpolation. The original and resampled data as well as a diff between them are written to one output NetCDF file. The unit tests include two known failure test cases where the expected result is not produced.
```

Comment:
this example set includes results output to a 4x4 array and a 2x2 array

as we may want to add more cases we may want to simply not include this level of resolution detail in the readme, instead writing something like

`These examples read in arbitrary data from a netCDF file and resample to a different horizontal domain as defined within the input netCDF file using bilinear interpolation`

for the first sentence?

Reply:
Thank you @mo-marqh - I think that's a good idea !